engine/src/main/java/org/camunda/bpm/engine/impl/ProcessEngineImpl.java

```diff
@@ -144,6 +145,10 @@ public ProcessEngineImpl(ProcessEngineConfigurationImpl processEngineConfigurati
   protected void executeSchemaOperations() {
     commandExecutorSchemaOperations.execute(processEngineConfiguration.getSchemaOperationsCommand());
     commandExecutorSchemaOperations.execute(processEngineConfiguration.getHistoryLevelCommand());
+    
+    if (processEngineConfiguration.isTelemetryEnabled()) {
```

Comment:
Let's remove this check and always insert the telemetry enabled property on engine startup. We probably also don't need to pass the flag to the command since we can access the engine config inside the command.